dqc_us_rules/dqc_us_0015.py

```diff
@@ -3,7 +3,7 @@
 # See PatentNotice.md for patent infringement notice.
 import csv
 import os
-from .util import facts, messages
+from .util import facts, messages, neg_num
```

Comment:
We are unable to use full paths in DQC because of the way plugins are managed in arelle.